ABOUT.md

```diff
@@ -1,9 +1,72 @@
+ * Shogun **runs natively under Linux/Unix, MacOS, and Windows**.
+ * Anyone can **try Shogun with minimal effort**: the Shogun [cloud](https://cloud.shogun.ml/hub/home) allows users to run a docker instance on Amazon’s AWS framework without any installation effort. All you need to do is to login via openID and connect to a jupyter notebook. Furthermore, we are continuously improving and simplifying the installation process for Shogun.
+ * Shogun offers **bindings to other sophisticated libraries including**: [LibSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/)/[LibLinear](http://www.csie.ntu.edu.tw/~cjlin/liblinear/), [SVMLight](http://svmlight.joachims.org/), [LibOCAS](http://cmp.felk.cvut.cz/~xfrancv/ocas/html/), [libqp](http://cmp.felk.cvut.cz/~xfrancv/libqp/html/), [VowpalWabbit](http://www.hunch.net/~vw/), [Tapkee](http://tapkee.lisitsyn.me/), [SLEP](http://www.public.asu.edu/~jye02/Software/SLEP/), [GPML](http://www.gaussianprocess.org/gpml/code/matlab/doc/) and more. See our [list of integrated external libraries](http://www.shogun-toolbox.org/page/about/contributions).
+
+### Free
+
+ * Shogun is **community based** and **non-commercial**.
+ * Licence: Shogun is currently released under GPLv3. We are working towards a BSD compatible repository and all newly added code is required to be BSD3 compliant.
+
+### Good (is what we aim for)
+
+ * Shogun has exclusive **state-of-the art algorithms** (among others: efficient SVM implementations, Multiple Kernel Learning, kernel hypothesis testing and Krylov methods). All of these are supported by a collection of general purpose methods for evaluation, parameter tuning, preprocessing, serialisation and I/O.
+ * Our implementation is **efficient and competitive** and we contribute to the benchmarking project by MLPack and Markus Edel (a framework to compare algorithms from different ML libraries).
+ * Shogun has an **extensive testing infrastructure**, making it reliable on dozens of OS setups.
+
+## <a name="useshoguntolearn"></a> Using Shogun to Learn
+
+Shogun is a community learning platform in every sense: **anyone can use the toolbox to learn about ML** and apply it to solve problems.
+
+ * Many **scientists** use Shogun as a tool for their research and it is [cited many hundred times on Google scholar](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=).
+ * We offer the Shogun cloud for **educational** purposes at universities and workshops (e.g. University College London GradSchool Introduction to Machine Learning).
```

Comment:
people outside London don't know that UCL stands for University College London. Also, "Introduction to ML" really suits our educational purpose. I would like to keep it (removed the GradSchool)